.github/workflows/cln-plugin.yaml

```diff
@@ -69,4 +73,4 @@ jobs:
       - name: Run tests
         run: |
           cd watchtower-plugin/tests
-          DEVELOPER=1 SLOW_MACHINE=1 poetry run pytest test.py --log-cli-level=INFO -s
+          VALGRIND=0 SLOW_MACHINE=1 poetry run pytest test.py --log-cli-level=INFO -s
```

Comment:
Is `VALGRIND` enabled by default?

Reply:
i think so, but not sure when and where it's used to be honest